code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -8,6 +8,7 @@
 	can_suppress = TRUE
 	burst_size = 1
 	fire_delay = 0
+	actions_types = list()
```

Comment:
If this is a non-modular change, please be sure to mark it as such.

Reply:
This is a change that tg has that we don't have for some reason

Reply:
As in, I directly copied the tg version of the file and then reapplied our edits to it